### ISSUE TITLE
Register "opengis" namespace into PyPI

### ISSUE THREAD

**Issue by desruisseaux** [opengeospatial/geoapi#40]
Python abstract classes are currently defined in the `opengis` namespace. This namespace (or any other namespace chosen by the working group) needs to be registered there:

* https://pypi.org/

Note: in a previous version we used the `ogc` namespace. But we switched to `opengis` because [the OGC acronym is used for many things](https://en.wikipedia.org/wiki/OGC) (Office of Government Commerce, Office of the General Counsel, _etc_). Because the Python community is larger than the geospatial community, the risk of name collision if using only "OGC" name seems high.

An alternative would be "opengeospatial". But this name is longer, and the Python community seems to aim for short names in general. The "opengis" name is closer to the historical name used by Java package, and also may be closer to the purpose of those interfaces since we are indeed proposing basic building block of a Geographic Information System (GIS).

**Comment by kinow** [comment]
What about `geoapi`?

**Comment by desruisseaux** [comment]
Could be. But "opengis.org" is a registered domain name by OGC (in addition of "opengeospatial.org"). I also noticed that OGC already uses the `org.opengis` namespace for other OGC developments in Java outside GeoAPI, for example test suite; I though that they could use a Python `opengis` namespace for non-GeoAPI stuff too.

**Comment by kinow** [comment]
Looks like it's trickier than I expected to find a good & available name. Just asked about `geoapi` as I noticed the Java artifacts were using it. But `opengis` looks good too.

**Comment by desruisseaux** [comment]
True, but `geoapi` is Maven artifact from "pre-Java 9" world. A Java 9 module name would be `org.opengis.something` if we follow the recommendation to stay close to package names. Maybe the module name would be `org.opengis.geoapi` - yes I agree it looks mixed, but is also consistent with the [complete standard name](http://www.geoapi.org/snapshot/standard_document.html) which is "OGC GeoAPI".
